tw-docops/tw-mkdocs/docs/index.md

```diff
@@ -1,38 +1,13 @@
-# Index Page
+# Libertad Pantoja
 
-## Wizeline DocOps Template
+## Technical Writer | Wizeline
 
-Welcome to the DocOps Template
+Libertad originally majored in Genomic Sciences where she was exposed to Big Data Analytics for the first time. After that, Libertad pursued a PhD in Biomedical Sciences. She previously worked in Data Science, focusing on banking data analytics. Her work also involved to transform the analytics portal in sata scientists friendly site.
```

Comment:
Libertad originally majored in Genomic Sciences, where she was exposed to Big Data Analytics for the first time. After that, Libertad pursued a Ph.D. in Biomedical Sciences. She previously worked in Data Science, focusing on banking data analytics. Her work also involved transforming the analytics portal into a data scientists-friendly site.

Reply:
some typos and a comma missing

Reply:
Thank you!